addons/mail/static/src/discuss/core/common/discuss_app_model_patch.js

```diff
@@ -0,0 +1,11 @@
+import { DiscussApp } from "@mail/core/common/discuss_app_model";
```

Comment:
Why moving discuss app in the wrong bundle rather than define this file in public_web instead?

Reply:
It's because the tabs are defined in `messaging_menu.xml`.

I could have patched the template, but we observe issues in defining models in non-common folder, therefore I moved the model to `common/` like we did with chatbot models.

Reply:
Only python models that we send on the bus are concerned by this issue, discuss app feels wrong to be elsewhere.

Other files should be fixed instead.

Reply:
Ok, I moved it back to `public_web`.

The issue was `appAsUnreadChannels`/`unreadChannels` that were defined in common, which lead to bundle issue on home page, preventing login page to open.